react/components/Modal/index.js

```diff
@@ -64,6 +65,7 @@ class Modal extends PureComponent {
         open={isOpen}
         little={centered}
         onClose={onClose}
+        onExited={onExited}
```

Comment:
```suggestion
        onExit={onExit}
```

Change to `onExit` to maintain consistence

Reply:
https://react-responsive-modal.leopradel.com/#props